docs/admin/cluster-management.md

```diff
@@ -66,37 +66,43 @@ Instance Group will take care of putting appropriate image on new machines and s
 
-The autoscaler will try to maintain the average CPU/memory utilization and reservation of nodes within the cluster close to the target value.
-The target value can be configured by `KUBE_TARGET_NODE_UTILIZATION` environment variable (default: 0.7) for ``kube-up.sh`` when creating the cluster.
-Node utilization is the total node's CPU/memory usage (OS + k8s + user load) divided by the node's capacity.
-Node reservation is the total CPU/memory requested by pods that are running on the node divided by the node's capacity.
-If the desired numbers of nodes in the cluster resulting from CPU/memory utilization/reservation are different,
-the autoscaler will choose the bigger number. The number of nodes in the cluster set by the autoscaler will be limited from `KUBE_AUTOSCALER_MIN_NODES` (default: 1)
-to `KUBE_AUTOSCALER_MAX_NODES` (default: the initial number of nodes in the cluster).
+Cluster autoscaler also scales down the cluster if it notices that some node is not needed anymore in a sence
+that:
 
-The autoscaler is implemented as a Compute Engine Autoscaler.
-The initial values of the autoscaler parameters set by `kube-up.sh` and some more advanced options can be tweaked on
-`Compute > Compute Engine > Instance groups > your group > Edit group`[Google Cloud Console page](https://console.developers.google.com)
-or using gcloud CLI:
+* It has more that 50% of cpu and memory not requested by any pod.
+* All of its pods can be easily restarted on the other nodes (they are replciated, don't have local storage) and there 
+  is no important system pods among them.
 
-```shell
-gcloud alpha compute autoscaler --zone $ZONE <command>
-```
+Cluster autoscaler is configured per instance group(GCE) or node pool(GKE). 
+
+If you are using GCE then you can either enable it while creating a cluster with kube-up.sh script or later with
+kube-push.sh. To configure cluser autoscaler you have to set 3 environment variables:
+
+* `KUBE_ENABLE_CLUSTER_AUTOSCALER` - it enables cluster autoscaler if set to true.
+* `KUBE_AUTOSCALING_MIN_NODES` - minimum number of nodes in the cluster.
+* `KUBE_AUTOSCALING_MAX_NODES` - maximum number of nodes in the cluster.
+
+On GKE you configure cluster autoscaler either on cluster creation or update or when creating a particular node pool
+(which you want to be autoscaled) by passing flags `--enable-autoscaling` `--min-nodes` and `--max-nodes`
+to the corresponding `gcloud` commands.
 
-Note that autoscaling will work properly only if node metrics are accessible in Google Cloud Monitoring.
-To make the metrics accessible, you need to create your cluster with `KUBE_ENABLE_CLUSTER_MONITORING`
-equal to `google` or `googleinfluxdb` (`googleinfluxdb` is the default value). Please also make sure
-that you have Google Cloud Monitoring API enabled in Google Developer Console.
+** Cluster autoscaler assumes that all nodes inside a signle instance group or node pool are the roughly the same,
```

Comment:
When viewing md doc this paragraph doesn't look any special. Maybe we should make it more visible. WDYT?